tests/nodescaling/README.md

```diff
@@ -51,32 +51,33 @@ terratest:
       controlplane: false
       worker: true
       quantity: 1
-  scaledUpNodepools:
-    - etcd: true
-      controlplane: false
-      worker: false
-      quantity: 3
-    - etcd: false
-      controlplane: true
-      worker: false
-      quantity: 2
-    - etcd: false
-      controlplane: false
-      worker: true
-      quantity: 3
-  scaledDownNodepools:
-    - etcd: true
-      controlplane: false
-      worker: false
-      quantity: 3
-    - etcd: false
-      controlplane: true
-      worker: false
-      quantity: 2
-    - etcd: false
-      controlplane: false
-      worker: true
-      quantity: 1
+  scalingInput:
```

Comment:
why is this duplicated in the root readme?

Reply:
The root README acts as the "one stop shop" for everything for the repo. The package READMEs more so reinforce select points if someone doesn't read the root README.